### DISCUSSION TITLE
Request: Does subview.xyz support Darwinia2?

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#9] [General]
It seems that it only support Crab2?

https://subview.xyz/

**Comment by 1022ranji** [comment]
Subview synced with Darwinia yesterday and they are having some problems adapting to Darwinia2 and need time to work on it.

**Comment by 1022ranji** [comment]
Subview now supports Darwinia2 https://darwinia.subview.xyz/
